src/widgets/FontSelector.tsx

```diff
@@ -0,0 +1,41 @@
+// SPDX-FileCopyrightText: © 2019 EteSync Authors
+// SPDX-License-Identifier: GPL-3.0-only
+
+import * as React from "react";
+import { ViewProps } from "react-native";
+import { Button, useTheme } from "react-native-paper";
+import { fontFamilies, FontFamilyKey } from "../helpers";
+import Menu from "./Menu";
+
+interface PropsType extends ViewProps {
+  visible: boolean;
+  selected: FontFamilyKey;
+  onChange: (item: string) => void;
```

Comment:
This is where the type is wrong (see my other comment), it should be: `item: FontFamilyKey`. After you fix this you can remove the other casts.